app/labor/sidekiq/worker_retries_exhausted_reporter.rb

```diff
@@ -0,0 +1,10 @@
+module Sidekiq
```

Comment:
as this is part of the initializer, shouldn't it be in `lib/sidekiq` with the other custom middleware?

Reply:
lol I love how I put things in lib you suggest they go to labor, I put things in labor you suggest they go to lib. Aye, one day I will get it right